### ISSUE TITLE
Tuning hyperparatmeters for the defended datasets

### ISSUE THREAD

**Issue by zxjbibobibobi** [robust-fingerprinting/RF#6]
Hello, I am trying to reproduce the experimental results of several WF attacks based on your implementations. Thank you for your awesome work.

I successfully ran the RF model on the undefended DF dataset, using the hyperparameters listed in the paper, and observed high metric scores.
Unfortunately, when I trained the model (with the same hyperparameters proved to be effective on the undefended dataset) on the DF dataset in the presence of the front defense, the model failed to converge over 30 epochs and performed extremely poorly on the testing data.
The same issue persists with other defenses like WTF-PAD and TrafficSliver,  
I wonder if it is necessary to adjust some hyperparameters, such as the learning rate. If yes, could you please provide the parameters used for evaluating the RF attack in the presence of specific defenses?

Thank you, sir!
